users/init.sls

```diff
@@ -1,5 +1,3 @@
-#!jinja|yaml
```

Comment:
I put it back in place, is this needed for Syntax highlighting in your Editor or does it serve another purpose?